### PR TITLE
Added white download icons

### PR DESCRIPTION
**Pull request by skjnldsv**
If we got white upload icons, we could really use some white download ones.
I'm in need of some for the contacts app, and I think it should be added in core for future possible uses.

Tell me if you think this is a bad idea :)
@owncloud/designers  @Henni

### CODE REVIEW

File: core/css/icons.css [hunk 1]
```diff
@@ -86,6 +86,10 @@
 	background-image: url('../img/actions/download.svg');
 }
```
Comment (jancborchardt) [inline core/css/icons.css]: Cut this line break, so it belongs to the icon-download. Otherwise nice!

**Comment by jancborchardt** [comment]
Nice! Looks like both are correctly minified/compressed, right?

**Comment by skjnldsv** [comment]
Well, i used the download icons black, revert the png and just added the fill parameter to the svg (like the upload-icon-white) Don't know if this could me more compressed :)

**Comment by jancborchardt** [comment]
Yup, looks good! :+1: 

Just squash the commits into one (the first, with the good commit message). :)

You can also in the future just amend your commits with:
```
git add your files
git commit --amend (keep the same commit message)
git push -f origin branch-name
```

**Comment by skjnldsv** [comment]
Without git rebase?

**Comment by jancborchardt** [comment]
Yeah, rebase is optional but different, just for updating it to current master. Amending and force pushing is for using the same commit in the current branch and overwriting it on the remote to not create two commits.

**Comment by skjnldsv** [comment]
Nice, I was always using a rebase HEAD to merge multiple commits. I only used ammend to correct an unpushed commit title.
Thanks!

**Comment by irgendwie** [comment]
LGTM :+1:

**Comment by skjnldsv** [comment]
let get this merged?

**Comment by skjnldsv** [comment]
@jancborchardt should we merge into 9.0.2 too?

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**